### PR TITLE
Added custom error message for validate_numeric and validate_size_of …

### PR DESCRIPTION
**Pull request by vishalsodani**
…#666

### CODE REVIEW

File: src/avram/validations.cr [hunk 1]
```diff
@@ -143,11 +144,11 @@ module Avram::Validations
       size = attribute.value.to_s.size
 
       if !min.nil? && size < min
-        attribute.add_error "is too short"
+        attribute.add_error message
```
Comment (matthewmcgarvey) [inline src/avram/validations.cr:147]: Can we try to keep the previous messages for both of these validations?

```suggestion
        attribute.add_error message || "is too short"
```

The message param would default to nil

**Review by matthewmcgarvey — CHANGES_REQUESTED** [review]

File: src/avram/validations.cr (hunk 1 above)
Comment (matthewmcgarvey) [inline src/avram/validations.cr:147]: These still need to have the same changes as you did below, then it will be good 👍

**Review by matthewmcgarvey — CHANGES_REQUESTED** [review]

**Review by matthewmcgarvey — APPROVED** [review]
Nice job! 🎉🎉🎉

**Comment by vishalsodani** [comment]
> Nice job! tadatadatada

Thanks!

**Comment by matthewmcgarvey** [comment]
@vishalsodani woops, didn't realize that the CI wasn't running. Could you go ahead and tell ameba to ignore those issues? I might try to figure out how to fix them later but it's overkill for this change

Here's docs on how to do it https://github.com/crystal-ameba/ameba#inline-disabling

**Comment by vishalsodani** [comment]
> @vishalsodani woops, didn't realize that the CI wasn't running. Could you go ahead and tell ameba to ignore those issues? I might try to figure out how to fix them later but it's overkill for this change
> 
> Here's docs on how to do it https://github.com/crystal-ameba/ameba#inline-disabling

I added the ignore comments.

**Comment by matthewmcgarvey** [comment]
Looks like the disabling didn't get put in the right place. You should be able to check it locally with `bin/ameba` (it's generated with `shards install`)

**Comment by vishalsodani** [comment]
> Looks like the disabling didn't get put in the right place. You should be able to check it locally with `bin/ameba` (it's generated with `shards install`)

Thanks for the hint. Fixed.

**Pull request merged**